test/e2e/tests/confirmations/signatures/permit.spec.ts

```diff
@@ -15,10 +15,6 @@ import { Ganache } from '../../../seeder/ganache';
 import { Driver } from '../../../webdriver/driver';
 
 describe('Confirmation Signature - Permit', function (this: Suite) {
-  if (!process.env.ENABLE_CONFIRMATION_REDESIGN) {
```

Comment:
There is only 1 test at the moment that needs it and I added it over there. `contract-interaction-redeisgn.spec`

Reply:
But this would still future-proof it for all new transaction tests to avoid that duplication?

Reply:
Looks like the current tests don't use the `withRedesignConfirmationFixtures` but set the preferences in the test itself, this would be a refactor I would like to address in a different PR if you agree, for now the test has the updated preference.